### PR TITLE
added snippets in random num generator

### LINKED ISSUE #47: Add snips in `src/RandomNumberGenerator.md` for other languages

### PR DESCRIPTION
**Pull request by MrSingh2000**
closes #47 
add snippets for random number generator for various languages including c-sharp, java, Go, etc.
No comments were made and function name and parameters were kept same.

### CODE REVIEW

**Review by rahulrk-dev — CHANGES_REQUESTED** [review]
Hi @MrSingh2000, just add function definitions instead of the whole program so anyone can just copy and use the code
And try to avoid any external libraries

**Comment by MrSingh2000** [comment]
Done

File: src/RandomNumberGenerator.md [hunk 1]
```diff
@@ -1,17 +1,128 @@
...
+  return Math.floor(Math.random() * (max - min - 1) + min + 1);
+}
+```
+
+### Cpp
+
+```cpp
+const unsigned long long a = 6364136223846793005ULL;
+const unsigned long long c = 1ULL << 30;
+const unsigned long long m = (1ULL << 62) - 1;
```
Comment (rahulrk-dev) [inline src/RandomNumberGenerator.md:35]: Keep this inside function, so the whole function can be copied easily

File: src/RandomNumberGenerator.md [hunk 2]
```diff
@@ -1,17 +1,128 @@
...
+
+    int random_number = min + (seed >> 31) % (max - min + 1);
+    return random_number;
+}
+```
+
+### Python
+
+```py
+def randomNumber(min_value, max_value, seed=None):
```
Comment (rahulrk-dev) [inline src/RandomNumberGenerator.md:50]: This function returns a single value every time instead of random, please check

**Review by rahulrk-dev — CHANGES_REQUESTED** [review]

File: src/RandomNumberGenerator.md (hunk 2 above)
Comment (MrSingh2000) [reply src/RandomNumberGenerator.md]: it was providing  a random value, but still added some more seed complications.

File: src/RandomNumberGenerator.md (hunk 1 above)
Comment (MrSingh2000) [reply src/RandomNumberGenerator.md]: done and updated some fixes

File: src/RandomNumberGenerator.md (hunk 2 above)
Comment (rahulrk-dev) [reply src/RandomNumberGenerator.md]: same thing mate
image

File: src/RandomNumberGenerator.md [hunk 3]
```diff
@@ -1,17 +1,118 @@
...
+    int random_number = min + (seed >> 31) % (max - min + 1);
+    return random_number;
+}
+
+```
+
+### Python
+
+```py
+def randomNumber(min_value, max_value):
```
Comment (rahulrk-dev) [inline src/RandomNumberGenerator.md:56]: Still return the same number mate
image

**Review by rahulrk-dev — CHANGES_REQUESTED** [review]

**Review by rahulrk-dev — APPROVED** [review]
Thank you @MrSingh2000, merging

**Pull request merged**